taca/analysis/analysis.py

```diff
@@ -303,17 +303,37 @@ def _process(run):
         # Previous elif might change the status to COMPLETED, therefore to avoid skipping
         # a cycle take the last if out of the elif
         if run.get_run_status() == 'COMPLETED':
+            run.check_run_status()
             logger.info(('Preprocessing of run {} is finished, transferring it'.format(run.id)))
             # Upload to statusDB if applies
             if 'statusdb' in CONFIG:
                 _upload_to_statusdb(run)
+                demux_summary_message = []
+                for demux_id, demux_log in run.demux_summary.items():
+                    if demux_log['errors'] or demux_log['warnings']:
+                        demux_summary_message.append("Sub-Demultiplexing in Demultiplexing_{} completed with {} errors and {} warnings:".format(demux_id, demux_log['errors'], demux_log['warnings']))
+                        demux_summary_message.append("\n".join(demux_log['error_and_warning_messages'][:5]))
+                        if len(demux_log['error_and_warning_messages'])>5:
+                            demux_summary_message.append("...... Only the first 5 errors or warnings are displayed for Demultiplexing_{}.".format(demux_id))
```

Comment:
Should there be a `break` here? Wouldn't this message be repeated many times otherwise?

Reply:
Ah right, maybe not.